cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -125,17 +97,122 @@ public void copy(ReadOnlyDataSource from, DataSource to) {
         if (from.exists(fromName)) {
             try (InputStream is = from.newInputStream(fromName);
-                    OutputStream os = to.newOutputStream(toName, false)) {
+                OutputStream os = to.newOutputStream(toName, false)) {
                 ByteStreams.copy(is, os);
             }
         }
     }
 
     private static final String FORMAT = "CGMES";
 
+    private static final String CHANGE_SIGN_FOR_SHUNT_REACTIVE_POWER_FLOW_INITIAL_STATE = "iidm.import.cgmes.change-sign-for-shunt-reactive-power-flow-initial-state";
+    private static final String CONVERT_BOUNDARY = "iidm.import.cgmes.convert-boundary";
+    private static final String CREATE_BUSBAR_SECTION_FOR_EVERY_CONNECTIVITY_NODE = "iidm.import.cgmes.create-busbar-section-for-every-connectivity-node";
+    private static final String POWSYBL_TRIPLESTORE = "iidm.import.cgmes.powsybl-triplestore";
+    private static final String STORE_CGMES_MODEL_AS_NETWORK_EXTENSION = "iidm.import.cgmes.store-cgmes-model-as-network-extension";
+    private static final String BOUNDARY_LOCATION = "iidm.import.cgmes.boundary-location";
+
+    private static final Parameter CHANGE_SIGN_FOR_SHUNT_REACTIVE_POWER_FLOW_INITIAL_STATE_PARAMETER = new Parameter(
+        CHANGE_SIGN_FOR_SHUNT_REACTIVE_POWER_FLOW_INITIAL_STATE,
+        ParameterType.BOOLEAN,
+        "Change the sign of the reactive power flow for shunt in initial state",
+        Boolean.FALSE)
+            .addAdditionalNames("changeSignForShuntReactivePowerFlowInitialState");
+    private static final Parameter CONVERT_BOUNDARY_PARAMETER = new Parameter(
+        CONVERT_BOUNDARY,
+        ParameterType.BOOLEAN,
+        "Convert boundary during import",
+        Boolean.FALSE)
+            .addAdditionalNames("convertBoundary");
+    private static final Parameter CREATE_BUSBAR_SECTION_FOR_EVERY_CONNECTIVITY_NODE_PARAMETER = new Parameter(
+        CREATE_BUSBAR_SECTION_FOR_EVERY_CONNECTIVITY_NODE,
+        ParameterType.BOOLEAN,
+        "Create busbar section for every connectivity node",
+        Boolean.FALSE)
+            .addAdditionalNames("createBusbarSectionForEveryConnectivityNode");
+    private static final Parameter POWSYBL_TRIPLESTORE_PARAMETER = new Parameter(
+        POWSYBL_TRIPLESTORE,
+        ParameterType.STRING,
+        "The triplestore used during the import",
+        TripleStoreFactory.defaultImplementation())
+            .addAdditionalNames("powsyblTripleStore");
+    private static final Parameter STORE_CGMES_MODEL_AS_NETWORK_EXTENSION_PARAMETER = new Parameter(
+        STORE_CGMES_MODEL_AS_NETWORK_EXTENSION,
+        ParameterType.BOOLEAN,
+        "Store the initial CGMES model as a network extension",
+        Boolean.TRUE)
+            .addAdditionalNames("storeCgmesModelAsNetworkExtension");
+    private static final String DEFAULT_BOUNDARY_LOCATION = PlatformConfig
+        .defaultConfig()
```

Comment:
I think a more consistent approach would be to use the `platformConfig` passed when `CgmesImport` is created:
```java
private final Parameter boundaryLocationParameter;

public CgmesImport(PlatformConfig platformConfig) {
        this.defaultValueConfig = new ParameterDefaultValueConfig(platformConfig);
        this.boundaryLocationParameter = new Parameter(
                BOUNDARY_LOCATION,
                ParameterType.STRING,
                "The location of boundary files",
                platformConfig.getConfigDir().resolve(FORMAT).resolve("boundary").toString());
    }

```

Reply:
Agree with your proposal, I will solve the conflicts with the support for post-processors that Geoffroy has added and commit in this branch